src/imgtools/dicom/crawl/parse_dicoms.py

```
@@ -466,3 +470,49 @@
+    -------
+        
+        list[dict[str, str]]
+            - the barebones dict with duplicates removed.    
+
+    Note: I tried to make this as efficient as possible, sorry if it slows down everything. 
+    """
+
+    hash_values = set()
+    output = []
+    for record in slim_db:
+        hash_key = hash("".join(["" if key == "SubSeries" else str(record[key]) for key in record.keys()]))
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)